src/align.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import {getPrefixedDataAttributes} from './utils/dom';
+import * as styles from './align.css';
+
+import type {DataAttributes} from './utils/types';
+
+type Props = {
+    x?: 'start' | 'center' | 'end';
+    y?: 'start' | 'center' | 'end';
```

Comment:
It is weird to me having props like `y` or `x`. I would expect something like `alignItems` or `justifyContent` like in `<Grid />` component. I'm looking for feature parity between those components

`<Grid alignItems="center" justifyItems="center"></Grid>`

https://mistica-web.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8UCWA3ABAZwC4E8AbGAXmGEwCMBDMAawHMAnCAVwDsp5MAdEZmPj6YAviIB8PdpkxIA4k1RRM1QqgbsAkrhgBbbCT5gY7HU2EALGOou5DIAIwAGJ%2BgvCAVqzyoAZvgBhCFMTOyNQmHMQSWkZWQDUJjBiHFQAL1JgABYnESpaRhYOKCDCCCYySDKmbAA6dhhWXCZVAAkbPIB6GJkkToUlGL60dBiQABoQXCtdGGwEAG0QAFkIdFQ8aiYQAF1JgHclafn4BYBmADYnHZEgA

Reply:
got it!